src/components/search/instructors/hiroko-nishimura/index.tsx

```diff
@@ -1,20 +1,146 @@
-import * as React from 'react'
+import React from 'react'
 import SearchInstructorEssential from '../instructor-essential'
-import HirokoNishimuraPageData from './hiroko-nishimura-page-data'
-import ResourceCta from 'components/search/instructors/resource-cta'
-import find from 'lodash/find'
+import Image from 'next/image'
+import {get} from 'lodash'
+import Link from 'next/link'
+import groq from 'groq'
+import {bpMinMD} from 'utils/breakpoints'
+import {track} from 'utils/analytics'
+import ExternalTrackedLink from 'components/external-tracked-link'
+import {HorizontalResourceCard} from 'components/card/horizontal-resource-card'
 
 export default function SearchHirokoNishimura({instructor}: {instructor: any}) {
-  const instructorData: any = find(HirokoNishimuraPageData, {
-    id: 'instructor-data',
-  })
+  const combinedInstructor = {...instructor}
+  const {courses, projects} = instructor
+  console.log(courses, projects)
+  const primaryProject = projects.resources
+  const [primaryCourse, secondaryCourse] = courses.resources
+  const location = 'Hiroko Nishimura instructor page'
 
-  const combinedInstructor = {...instructor, ...instructorData}
+  return (
+    <div>
+      <SearchInstructorEssential
+        instructor={combinedInstructor}
+        CTAComponent={
+          <FeaturedCourse resource={primaryCourse} location={location} />
+        }
+      />
+      <section className="flex md:flex-row flex-col max-w-screen-xl mx-auto gap-3 px-5 md:px-0 -mt-10">
+        <HorizontalResourceCard
+          resource={primaryProject}
+          location={location}
+          className="md:w-2/5"
+        />
+        <HorizontalResourceCard
+          resource={secondaryCourse}
+          location={location}
+          className="md:w-3/5"
+        />
+      </section>
+    </div>
+  )
+}
+
+export const hirokoNishimuraQuery = groq`*[_type == 'resource' && slug.current == "hiroko-nishimura-landing-page"][0]{
+  'courses': resources[slug.current == 'instructor-landing-page-featured-courses'][0]{
+    resources[]->{
+      title,
+      'description': summary,
+    	path,
+      byline,
+    	image,
+      'background': images[label == 'feature-card-background'][0].url,
+      'instructor': collaborators[]->[role == 'instructor'][0]{
+      	'name': person->.name
+    	},
+    }
+  },
+	'projects': resources[slug.current == 'instructor-landing-page-projects'][0]{
+    resources[0]{
+      title,
+      'path': url,
+      path,
+      description,
+      image,
+      byline
+    }
+  }
+}`
 
+const FeaturedCourse: React.FC<{location: string; resource: any}> = ({
```

Comment:
@dealingwith this page uses a one-off component so those darkmode style issues should be isolated to this page.

@lsminter here's an example of how to fix this
https://github.com/eggheadio/egghead-next/blob/main/src/components/search/instructors/stephanie-eckles/index.tsx#L186-L206